nautilus_trader/indicators/helpers.pyx

```diff
@@ -0,0 +1,62 @@
+cdef bint cross_over(str direction, list values1, list values2) except *:
+    """
+    Determines if the values in list1 crossed list2 in the direction specirfied.
+
+    Parameters
+    ----------
+    direction : str
+        The direction of the cross over to check. Accepts 'UP' or 'DOWN'.
+    values1 : list
+        The first list of values.
+    values2 : list
+        The second list of values.
+
+    Returns
+    -------
+    bint
+
+    """
+    if direction == "UP":
+        return cross_up(values1, values2)
+    elif direction == "DOWN":
+        return cross_down(values1, values2)
```

Comment:
We could make these an Enum.

Reply:
Yes potentially, or if there are ever only two possible values a bool (`bint`) would work also.

Reply:
Were you planning on adding further indicators which use this `cross_over` function? I'd suggest until then we keep this function with the `LinearRegression` code.

Reply:
I'm not entirely sure where to move these to. They're helpers that can be used directly in indicators or in strategies. I do plan on using them more, but if everyone doesn't find them useful, can remove it from PR and keep it local.

Reply:
You could move the function into the same file, that's where I would move it for now.

Reply:
I will remove the helpers from this PR, that way we can keep it independent of the indicator logic. I will just use it in a local file.